server/server.go

```diff
@@ -51,19 +71,44 @@ func (fs *FurtiveServer) connectionHandler(w http.ResponseWriter, r *http.Reques
 
 	// Register client and send initial message to client
 	fs.registerClient(ws)
-	fs.sendMessageToClient(&Message{fs.question}, ws)
+	
+	// TODO: Necessary to add id for clients (now 0) -> FROM 0, not 1
+	data := &VotingData{0, fs.question, big.NewInt(3), big.NewInt(3863), big.NewInt(7727)}
+	fs.sendMessageToClient(&Message{"votingData", data}, ws)
 
 	// Pass messages from clients to other clients
+	// TODO: Now only one client
 	for {
-		var msg *Message
+		var contents json.RawMessage
+		msg := &Message{
+			Contents: &contents,
+		}
 		if err := ws.ReadJSON(&msg); err != nil {
 			log.Error("Error when reading message from client:", err)
 			fs.removeClient(ws)
 			ws.Close()
 			return
 		}
-		fs.broadcastChan <- msg
-		log.Info("Received message from client: ", msg.Contents)
+		log.Info("New message type: ", msg.Type)
+
+		switch msg.Type {
+		case "roundOne":
+			var value *Value
+			if err := json.Unmarshal(contents, &value); err != nil {
+				log.Fatalln("Error when reading JSON:", err)
+				return
+			}
+			fs.sendMessageToClient(&Message{"roundOne", &Values{[]*big.Int{value.Number}, 1}}, ws)
+		case "roundTwo":
+			var value *Value
+			if err := json.Unmarshal(contents, &value); err != nil {
+				log.Fatalln("Error when reading JSON:", err)
+				return
+			}
+			fs.sendMessageToClient(&Message{"roundTwo", &Values{[]*big.Int{value.Number}, 1}}, ws)
```

Comment:
Aren't this both cases equal to each other? This switch could be unified to single if